Functions/Initialization/xASL_init_Import.m

```diff
@@ -65,4 +65,6 @@
     % Create logging directory if it does not exist already
     xASL_adm_CreateDir(fullfile(x.dir.DatasetRoot,'derivatives','ExploreASL','log'));
 
+	x = xASL_init_LoadDataPar(x);
```

Comment:
This needs reverting: dataPar is for processing, not for import. We strictly separated this and probably even removed this init from here, remember?

Reply:
Agree